app/main.py

```diff
@@ -1,3 +1,75 @@
 class Distance:
-    # Write your code here
-    pass
+    """
+    Magic methods
+    """
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return "Distance(km=" + str(self.km) + ")"
+
+    def __add__(self, other: int) -> "Distance":
```

Comment:
`other` can be float and `Distance`. Fix type hint here and in other functions too